docs/technical_reference/unit_models/energy_models/pv_surrogate.rst

```diff
@@ -0,0 +1,34 @@
+Photovoltaic (PV) Surrogate Model
+====================================================
+
+This Photovoltaic (PV) unit model
+   * supports steady-state only
+   * is a surrogate model
+   * is created using data from the `System Advisor Model (SAM) <https://sam.nrel.gov>`_
+
+.. TODO: Add index/reference to home page
+
+
+Degrees of Freedom
+------------------
+The PV model has 1 degree of freedom that should be fixed for the unit to be fully specified.
+
+Model Structure
+---------------
+
+This PV Surrogate model is created using data from the SAM tool and is trained using PySMO RBF functions.
```

Comment:
Sort of pedantic but should we specify that the data is gathered using PySAM, not SAM? Just so a potential future user isn't trying to figure out how to get SAM to give them the data they need (this might be possible but I don't know for sure)

Reply:
Fixed